src/coreclr/jit/emitarm64.cpp

```diff
@@ -22225,13 +22389,48 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             break;
 
         case IF_SVE_DZ_1A: // ........xx...... .............DDD -- sve_int_pn_ptrue
+        case IF_SVE_EA_1A: // ........xx...... ...iiiiiiiiddddd -- SVE broadcast floating-point immediate (unpredicated)
+        case IF_SVE_EC_1A: // ........xx...... ..hiiiiiiiiddddd -- SVE integer add/subtract immediate (unpredicated)
             result.insThroughput = PERFSCORE_THROUGHPUT_2C;
             result.insLatency    = PERFSCORE_LATENCY_2C;
             break;
 
-        case IF_SVE_EA_1A: // ........xx...... ...iiiiiiiiddddd -- SVE broadcast floating-point immediate (unpredicated)
-            result.insThroughput = PERFSCORE_THROUGHPUT_2C;
-            result.insLatency    = PERFSCORE_LATENCY_2C;
+        case IF_SVE_ED_1A: // ........xx...... ...iiiiiiiiddddd -- SVE integer min/max immediate (unpredicated)
+            switch (ins)
+            {
+                case INS_sve_umin:
+                    result.insThroughput = PERFSCORE_THROUGHPUT_1C; // need to fix
+                    result.insLatency    = PERFSCORE_LATENCY_1C;    // need to fix
+                    break;
+                default:
+                    result.insThroughput = PERFSCORE_THROUGHPUT_2C;
+                    result.insLatency    = PERFSCORE_LATENCY_2C;
+                    break;
+            }
+            break;
+
+        case IF_SVE_EE_1A: // ........xx...... ...iiiiiiiiddddd -- SVE integer multiply immediate (unpredicated)
+            result.insThroughput = PERFSCORE_THROUGHPUT_2X;
+            result.insLatency    = PERFSCORE_LATENCY_5C;
+            break;
+
+        case IF_SVE_EB_1A: // ........xx...... ..hiiiiiiiiddddd -- SVE broadcast integer immediate (unpredicated)
+            switch (ins)
+            {
+                // TODO-SVE: Why are these different? MOV is an alias for DUP
```

Comment:
I forgot to mention this above, but should these PerfScore values be different? If so, if the instruction is `INS_sve_dup` in `emitIns_R_I`, we change it to `INS_sve_mov` since that is the preferred disassembly. That means the `INS_sve_dup` case is unreachable when determining PerfScores. I can change my logic so we set the instruction to `INS_sve_dup` in `emitIns_R_I`, and then print the instruction as a `mov` in `emitDispInsHelp`, but that will require introducing some edge case logic into the latter method, as we currently don't do anything special for printing aliased instructions. 

If these PerfScores should be the same, then we can avoid that issue altogether.

Reply:
The alias isn't changing the encoding at all. All the bits in the instruction are the same, it's just the way it's printed that changes.

Looking at the table [here](https://github.com/dotnet/runtime/issues/94549#issuecomment-1828434944)
Mov and dup exist in multiple rows:

Broadcast logical bitmask immediate to vector | DUPM, MOV | 2 | 2 | V | -
-- | -- | -- | -- | -- | --
Duplicate, immediate and indexed form | DUP, MOV | 2 | 2 | V | -
Duplicate, scalar form | DUP, MOV | 3 | 1 | M0 | -

I think you should be using the first one (2,2) for all of IF_SVE_EB_1A.

Was this incorrect in the boiler plate autogenerated code?

Reply:
I see, thank you for clarifying. Yes, the boilerplate code has different PerfScore values for `mov` and `dup` for the `IF_SVE_EB_1A` format -- it looks like the generator tool mixed up the PerfScore values from different formats. I'll update this to use (2,2).

Reply:
>it looks like the generator tool mixed up the PerfScore values from different formats

@kunalspathak something to fix